src/dex-ui/pages/dao/DAOProposalVoting.tsx

```diff
@@ -16,8 +16,15 @@ export const DAOProposalVoting = (props: DAOProposalVotingProps) => {
     turnout = Math.round((proposal.approvalCount / dao.threshold) * 100);
   }
   let votingEndTime = "";
-  if (isMultiSig) {
-    votingEndTime = proposal.timestamp;
+  if (proposal.timestamp) {
+    const date = new Date(Math.floor(+proposal.timestamp));
+    votingEndTime = date.toLocaleString([], {
+      day: "2-digit",
+      month: "2-digit",
+      year: "numeric",
+      hour: "2-digit",
+      minute: "2-digit",
+    });
   }
```

Comment:
we could use `votingEndTime  =  proposal.timestamp  ? getFormattedEndTime(proposal.timestamp)  : ""`
where `getFormattedEndTime` would be a util function to be used in other places in future if required.

Reply:
Moved the Time formatting into time utils.